src/devices/DimmableLightingDevice/DimmableLightingDevice.ts

```diff
@@ -216,6 +361,38 @@ export default class DimmableLightingDevice extends InsteonDevice {
 		return this.sendInsteonCommand(0x18,0x00);
 	}
 	
+	/* Not yet tested. Based soley on documentation */
```

Comment:
Will test all three of these as soon as possible - I believe the last batch of dimmer switches that I bought support the light on at rate commands.

Reply:
I can test these on my dimmers if needed